zivid_camera/test/test_zivid_camera.cpp

```diff
@@ -460,6 +465,69 @@ TEST_F(CaptureOutputTest, testCapturePointsXYZ)
   }
 }
 
+#if ZIVID_CORE_VERSION_MAJOR >= 2 && ZIVID_CORE_VERSION_MINOR >= 9
```

Comment:
This tests looks incorrect, as it would break if we go to 3.0.

Reply:
can't we fix it after 3.0? I am not sure if we committed to 3.0 including all API breaking changes or just 2.10 for the next release

Reply:
I think we must fix this so it works as expected in the future. It could be that a 3.10 does not require much changes in ROS driver (if any). It should be quick to fix.

Reply:
It could be that we switch to a monotonic version number like https://github.com/llvm/llvm-project/blob/466d554dcab39c3d42fe0c5b588b795e0e4b9d0d/clang/lib/Headers/stdatomic.h#L56 for API 3.0 and not use split major-minor versions for this check?

Reply:
anyway, I added one more check for 3.0